databricks/koalas/groupby.py

```diff
@@ -2034,12 +2083,16 @@ def nunique(self, dropna=True):
         Name: value1, dtype: int64
         """
         if dropna:
-            stat_function = lambda col: F.countDistinct(col)
+
+            def stat_function(col):
+                return F.countDistinct(col)
+
         else:
-            stat_function = lambda col: (
-                F.countDistinct(col)
-                + F.when(F.count(F.when(col.isNull(), 1).otherwise(None)) >= 1, 1).otherwise(0)
-            )
+
+            def stat_function(col):
+                return F.countDistinct(col) + F.when(
+                    F.count(F.when(col.isNull(), 1).otherwise(None)) >= 1, 1
+                ).otherwise(0)
```

Comment:
ditto ?

Reply:
Oh maybe It could have changed automatically when running `reformatter`, right ?

Reply:
Yes, all of that was when running the reformatter. I can try to drop these changes, where it was to unrelated places in this change and just correct the issue in the new code, that was added here.

Reply:
All those non-related changes should be gone in this PR now